src/vs/code/electron-main/app.ts

```diff
@@ -163,29 +163,89 @@ export class CodeApplication extends Disposable {
 		// !!! DO NOT CHANGE without consulting the documentation !!!
 		//
 
+		const isFromCore = (requestingUrl?: string | undefined) => requestingUrl?.startsWith(`${Schemas.vscodeFileResource}://${VSCODE_AUTHORITY}`);
```

Comment:
`isUrlFromWindow`? Aligns with following variable and uses the more familiar "window" proc, I normally refer to the repo as "core".